set.mm

```diff
@@ -26836,8 +26845,21 @@ generally appear in a single form (either definitional, but more often
   ${
     $d x A y $.  $d ph y $.
     $( Membership of a class variable in a class abstraction.  (Contributed by
-       NM, 23-Dec-1993.)  (Proof shortened by Wolf Lammen, 16-Nov-2019.) $)
+       NM, 23-Dec-1993.)  (Proof shortened by Wolf Lammen, 16-Nov-2019.)  Avoid
+       ~ ax-11 , using the proof of ~ sbc5ALT .  (Revised by SN,
```

Comment:
What do you mean by "using the proof of ~ sbc5ALT " ? Do you mean "using ideas from the proof of ~ sbcALT" ? If yes, then the latter is a clearer formulation.

**edit:** if we obsolete the latter, then we should write:
> ... using ideas used by NM and MC in a fomer proof of ~ sbc5 .

in order to keep the credit to NM and MC.

Reply:
I meant the former; the proofs are in a sense, the same. But, in a sense, so to clarify the equivalence I expanded the wording.